tests/golos.publication_tests.cpp

```diff
@@ -200,17 +187,16 @@ BOOST_FIXTURE_TEST_CASE(set_params, golos_publication_tester) try {
     params = "[" + post.get_str_comment_depth(0) + "]";
     BOOST_CHECK_EQUAL(err.max_cmmnt_dpth_less_0, post.set_params(params));
     
-    params = "[" + post.get_str_social_acc(name()) + "]";
+    params = "[" + post.get_str_social_acc(N(gls.social)) + "]";
     BOOST_CHECK_EQUAL(err.no_social_acc, post.set_params(params));
     
-    params = "[" + post.get_str_referral_acc(name()) + "]";
+    params = "[" + post.get_str_referral_acc(N(gls.referral)) + "]";
```

Comment:
it's better to have this names somewhere in config. this will help a lot in case of changing default